events/triggers/passwd

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+signal="${1}"
```

Comment:
Passing password on command line is not great but not too bad given current assumptions. Make sure to add this to your TODO list of things to fix when implementing isolation.